libs/multi-chain/src/lib/near/account.ts

```diff
@@ -84,20 +84,32 @@ export class NearAccount {
     return this.nearSingleton;
   }
 
-  getAccountNameFromPubkey(pubkey: PublicKey, curve: CryptoCurves) {
+  // TODO: get info from smart contract
+  async getAccountNameFromPubkey(secpPK: PublicKey, curve: CryptoCurves) {
     return NearAccount.getAccountNameFromPubkey(
-      pubkey,
+      secpPK,
       curve,
       this.params.connectConfig.networkId as NearNetworkId
     );
   }
 
-  static getAccountNameFromPubkey(
+  async setAccountName(
+    edPK: PublicKey,
+    edSig: string,
+    secpPK: PublicKey,
+    secpSig: string,
+    newAccountID: string,
+    msg: string
+  ) {
+    // TODO: stubbed
```

Comment:
Just to make understand what this is doing, this function is to call the contract?

Reply:
This is deleted in the next PR